ntoskrnl/mm/mminit.c

```diff
@@ -45,7 +45,7 @@ extern NTSTATUS MiRosTrimCache(ULONG Target, ULONG Priority, PULONG NrFreed);
 VOID
 INIT_FUNCTION
 NTAPI
-MiCreateArm3StaticMemoryArea(PVOID BaseAddress, ULONG Size, BOOLEAN Executable)
+MiCreateArm3StaticMemoryArea(PVOID BaseAddress, SIZE_T Size, BOOLEAN Executable)
```

Comment:
Right.
Nit:
[MmCreateMemoryArea grep](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=MmCreateMemoryArea)
mm.h has `SIZE_T`, but marea.c has `ULONG_PTR`: should the latter be syn'ed too?

Reply:
First of all it doesn't matter at all, since ULONG_PTR and SIZE_T are technically the same. I also suggest not putting too much "nits" into PR reviews (that's 90% of my PRs at work and I hate it), since we don't want to turn providing an improvement into an obligation to also "fix" the rest of our code. Think about why there is Wine-Staging and why we use it. We don't have Ros-Staging so it could become dev/null at some point. Let's not go there.